crates/viewer/re_ui/data/light_theme.ron

```diff
@@ -5,115 +5,114 @@
   "small_icon_size": 14,
 
   "Alias": {
-    "native_frame_stroke": {
-      "color": "{Gray.750}",
-      "width": 1
+    "native_frame_stroke": { //couldn't find
```

Comment:
This is currently unused, but is part of a feature to make Rerun look less ugly on Windows and Linux: https://github.com/rerun-io/rerun/issues/1063